### DISCUSSION TITLE
Update field on create marker

### DISCUSSION THREAD

**Discussion by fizzi** [Budibase/budibase#8021] [Help]
Hi all, 

i am trying to update a field when a marker is created on the map. I defined 2 actions which updated two text fields. I also tried with numbers.

![Schermata 2022-09-28 alle 15 44 56](https://user-images.githubusercontent.com/444291/192795330-326d9ef9-e308-478f-b7df-f78764125033.png)

![Schermata 2022-09-28 alle 15 45 11](https://user-images.githubusercontent.com/444291/192795388-c4a98e96-a22f-48d2-a629-91d9608d3de7.png)

When I create the marker on the map, no fields are updated.

Am I wrong something?

Thanks,

**Comment by fizzi** [comment]
Hi all,

analyzing the problem and inspecting the html flow, as you can see in the image, when the marker is created, the html code for the buttons is added but they are not visible and not clickable. I think it's a bug.

![Schermata 2022-09-29 alle 11 15 00](https://user-images.githubusercontent.com/444291/192992964-9c71adfb-a159-40e0-a386-26152185e610.png)

the css style that is applied is:

`.button-container {
    display: flex;
    flex-direction: row;
    justify-content: center;
    align-items: center;
    gap: var(--spacing-xl);
    margin-top: var(--spacing-xl); <- this i think is the problem
  }`

Thanks

**Reply by JDiego199** [reply]
hello @fizzi 
I have the same error, were you able to solve it?

**Comment by melohagan** [comment]
Hi @fizzi 

I'm not seeing the issue you're describing:
![map](https://user-images.githubusercontent.com/101575380/194816314-e81d83d6-de52-4f04-b73b-3a9c16f256bc.gif)

<img width="1156" alt="Screenshot 2022-10-10 at 08 27 23" src="https://user-images.githubusercontent.com/101575380/194816536-cd7c20b5-237a-40cb-be8e-02ea53c4afa8.png">
